src/celengine/univcoord.h

```diff
@@ -81,9 +85,9 @@ class UniversalCoord
     Eigen::Vector3f offsetFromLy(const Eigen::Vector3f& v) const
     {
         Eigen::Vector3f vUly = v * 1.0e6f;
-        Eigen::Vector3f offsetUly((float) (x - (BigFix) vUly.x()),
-                                  (float) (y - (BigFix) vUly.y()),
-                                  (float) (z - (BigFix) vUly.z()));
+        Eigen::Vector3f offsetUly(static_cast<float>(x - static_cast<BigFix>(vUly.x())),
```

Comment:
all these static_casts make my eyes bleed. could you introduce methods `toFloat()` & `toDouble` as we have `toString()` and use the instead?
```
Eigen::Vector3f offsetUly((x - BigFix(vUly.x()).toFloaf(),...)
```

Reply:
In this case, I can replace the `static_cast<BigFix>` with just a constructor call, it looks a bit neater now. I'd rather not duplicate functionality between the operators (used in various other places in the code base) and methods